spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Not really. SF code is more ugly than we expect: https://github.com/spring-projects/spring-framework/blob/master/spring-core/src/test/java/org/springframework/core/ConventionsTests.java.

They allow to use *asterisk* !!! :open_mouth: